components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/FirefoxAccount.kt

```diff
@@ -101,15 +102,15 @@ class FirefoxAccount internal constructor(
 
     override fun beginOAuthFlowAsync(scopes: Set<String>) = scope.async {
         handleFxaExceptions(logger, "begin oauth flow", { null }) {
-            val url = inner.beginOAuthFlow(scopes.toTypedArray())
+            val url = inner.beginOAuthFlow(scopes.toTypedArray(), "notset")
             val state = Uri.parse(url).getQueryParameter("state")!!
             AuthFlowUrl(state, url)
         }
     }
 
     override fun beginPairingFlowAsync(pairingUrl: String, scopes: Set<String>) = scope.async {
         handleFxaExceptions(logger, "begin oauth pairing flow", { null }) {
-            val url = inner.beginPairingFlow(pairingUrl, scopes.toTypedArray())
+            val url = inner.beginPairingFlow(pairingUrl, scopes.toTypedArray(), "notset")
```

Comment:
Will create an issue for this, probably going to require upstream consumers to set the `entrypoint`.

Reply:
Right. Adding a bit of context: FxA basically considers any attempt to load the login page without an `entrypoint` parameter to be a metrics bug, as we have no way of attributing the signin metrics for that flow back to a particular UI touchpoint. Making this a required argument surfaces that requirement to the application, so we probably don't want to paper over it at this level. (Although @grigoryk might prefer to paper over it here for a short period while considering the API implications).

Reply:
If we do decide to temporarily hard-code a default entrypoint here, I suggest the string "none"; this is what FxA uses internally when you don't provide an explicit entrypoint value.